sqld/src/heartbeat.rs

```diff
@@ -13,10 +13,7 @@ pub async fn server_heartbeat(
     let client = reqwest::Client::new();
     loop {
         sleep(update_period).await;
-        let body = StatsResponse {
-            rows_read_count: stats.rows_read(),
-            rows_written_count: stats.rows_written(),
-        };
+        let body: StatsResponse = (&stats).into();
```

Comment:
nit: in that specific case I find this more readable:
```suggestion
        let body = StatsResponse::from(&stats);
```

Reply:
indeed!